src/commonMain/kotlin/exchange.dydx.abacus/calculator/TriggerOrdersInputCalculator.kt

```diff
@@ -89,13 +90,14 @@ internal class TriggerOrdersInputCalculator(val parser: ParserProtocol) {
         val positionSide = parser.asString(parser.value(position, "resources.indicator.current"))
         val positionSize = parser.asDouble(parser.value(position, "size.current"))?.abs() ?: return modified
         val notionalTotal = parser.asDouble(parser.value(position, "notionalTotal.current")) ?: return modified
-        val leverage = parser.asDouble(parser.value(position, "leverage.current"))?.abs() ?: return modified
+        val leverage = requireNotNull(parser.asDouble(parser.value(position, "leverage.current"))) { "leverage was null" }
```

Comment:
chatted w @prashanDYDX  let's revert `requireNotNull`

it's not with 100% certainty that `leverage.current` is non-null and `requireNotNull` throws an error which would break in semi-valid cases.